stac_ipyleaflet/core.py

```diff
@@ -64,6 +60,16 @@ def __init__(self, **kwargs):
         self.inspect_widget = None
         self.marker_added = False
 
+        self.create_load_gif()  # @QUESTION: Is this even used though? Can we just get rid of it?
+        self.create_buttons_layout()
+        self.add_biomass_layers_options()
+        self.add_custom_tools()
+
+        self.point_control = InspectControlWidget.template(self)
+        self.draw_control = DrawControlWidget.template(self)
+        return None
+
+    def create_load_gif(self):
```

Comment:
The `create_load_gif` function to remove.

Reply:
🙌🏼 I'm glad you would like to remove as well. Some people like to keep unused code in the codebase even if it is not being used to reference while others have a strong opinion on removing. I personally like to remove code that is not being used at all because there is no point for it to live there in my opinion. Github will have have the history so if anything, branches or previous commits could be referenced to get the same code back